pos_order_split_invoice_check_birthdate/views/account_move_views.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_move_form_split_birthdate_check" model="ir.ui.view">
+        <field name="name">view.move.form.split.birthdate.check</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="pos_order_split_invoice.account_move_form_view" />
+        <field name="arch" type="xml">
+            <field name='splitting_partner_id' position="after">
+                <field
+                    name='splitting_partner_birthdate'
+                    invisible="not splitting_order_id"
+                />
```

Comment:
it would be amazing if you can show the age between brackets
ie : (34 y) --> (34 a) in french ;-)